core/src/main/scala/kafka/server/KafkaRaftServer.scala

```diff
@@ -100,6 +101,9 @@ class KafkaRaftServer(
       controllerQuorumVotersFuture
     ))
   } else {
+    // we need to register the various kafka.controller metrics
+    // for backwards compatibility with the ZooKeeper-based case
+    new QuorumControllerMetrics(KafkaYammerMetrics.defaultRegistry())
```

Comment:
This works because the constructor of `QuorumControllerMetrics` registers those metrics against the Yammer registry. How about having a static method in `QuorumControllerMetrics` that does this and doesn't use guages since they are not needed?

Reply:
It doesn't look like there is a way to register default metrics in the Yammer metrics library.

Reply:
> How about having a static method in QuorumControllerMetrics that does this and doesn't use gauges since they are not needed 

Yeah, I don't see a way that this can be done.  The registry requires that we add a Meter, Counter, Histogram, Timer, or Gauge.  The Gauges don't take up a lot of resources anyway, so it doesn't seem like a big deal.